packages/Webkul/Admin/src/Resources/views/components/layouts/header/index.blade.php

```diff
@@ -167,6 +178,163 @@ class="text-sm {{ $menu->getActive($subMenuItem) ? 'text-violet-700 dark:text-vi
+                    >
+                        @lang('admin::app.notifications.view-all')
+                    </a>
+
+                    <a
+                        class="text-xs text-violet-700 font-semibold cursor-pointer transition-all hover:underline"
+                        v-if="userNotifications?.length"
+                        @click="readAll()"
+                    >
+                        @lang('admin::app.notifications.read-all')
+                    </a>
+                </div>
+            </x-slot>
+        </x-admin::dropdown>
+    </script>
+
+    <script type="module">
+        app.component('v-notifications', {
+            template: '#v-notifications-template',
+
+                props: [
+                    'getReadAllUrl',
+                    'readAllTitle',
+                ],
+
+                data() {
+                    return {
+                        userNotifications: [],
+
+                        totalUnRead: 0,
+                    }
+                },
+
+                mounted() {
+                    this.getNotification();
+                },
+
+                methods: {
+                    getNotification() {
+                        this.$axios.get('{{ route('admin.notification.get_notification') }}', {
+                                params: {
+                                    limit: 5,
+                                    read: 0
+                                }
+                            })
+                            .then((response) => {
+                                this.userNotifications = response.data.search_results.data;
+
+                                this.totalUnRead =   response.data.total_unread;
+                            })
+                            .catch(error => console.log(error))
+                    },
+
+                    timeAgo(timestamp) {
+                        const now = new Date();
+                        const past = new Date(timestamp);
+                        const diffInSeconds = Math.floor((now - past) / 1000);
+
+                        const intervals = {
+                            year: 365 * 24 * 60 * 60,
+                            month: 30 * 24 * 60 * 60,
+                            week: 7 * 24 * 60 * 60,
+                            day: 24 * 60 * 60,
+                            hour: 60 * 60,
+                            minute: 60,
+                            second: 1
+                        };
+
+                        for (const [unit, secondsInUnit] of Object.entries(intervals)) {
+                            const count = Math.floor(diffInSeconds / secondsInUnit);
+                            if (count > 0) {
+                                return count === 1 ? `${count} ${unit} ago` : `${count} ${unit}s ago`;
+                            }
+                        }
+
+                        return "just now";
```

Comment:
add translation here

Reply:
Resolved:
```
$this->created_at->diffForHumans()
```

This method provides a human-readable, localized value based on the application's current locale settings